### DISCUSSION TITLE
Công suất DC Motor hoạt động không ổn định

### DISCUSSION THREAD

**Discussion by MastodonRoboticsTeam** [MastodonRoboticsTeam/VRC2023#10] [Q&A] [answered]
Motor khi kéo cần Analog (Joystick) trên Controller PS2 có hiện tượng giật lại và chạy rất chậm

**Comment by ghost** [comment, marked as answer]
Lỗi chủ yếu "có thể" là do speed khi khai vận tốc cực đại hoặc cực tiểu sai

Ví dụ trong trường hợp khai tốc độ theo milisecond nếu lỡ chỉnh quá thành 4096 thay vì 4095 theo xung của PWM (Xung PWM có giới hạn từ 0 -> 4095) thì hiện tượng chạy quá nhanh hay chậm ở một phía hoàn toàn có thể xảy ra

Code sai:

```
#define SPD_MAX 4096
```

Giải pháp để sửa lại:

```
#define SPD_MAX 4095
```
